main.py

```diff
@@ -1,125 +1,118 @@
 #### Imports ####
 #################
-
+import os.path
 from datetime import datetime
+from pathlib import Path
+
+from ba_infra.presto_connection import WixPrestoConnection
+from google.oauth2 import service_account
 
 import functions as func
 
-###########################
-#### Global Parameters ####
-###########################
+#####################
+####  Parameters ####
+#####################
+DATA_FOLDER = Path(os.getcwd())
 
-api_key = # Upon production the key will be taken from airflow storage, during development the key is taken from text file
+BQ_CREDENTIALS = service_account.Credentials.from_service_account_file(DATA_FOLDER / "key.json", scopes=["https://www.googleapis.com/auth/cloud-platform"])
+API_KEY = func.read_json(path_file=DATA_FOLDER / "key_api.json")['api_key']
+
+GLOBAL_PARAMETERS = func.read_json(path_file=DATA_FOLDER / "configuration.json")
 
 # Set dates
-date_mid_month = func.mid_month()   # Data updated two weeks after the end of the month, for the previous month
-current_time = datetime.now()
+DATE_MID_MONTH = func.mid_month()   # Data updated two weeks after the end of the month, for the previous month
+CURRENT_TIME = datetime.now()
+
+# Set Tables
+GOOGLE_SHEET_KW_TABLE = 'gdrive.bi.marketing_itaib_semrush_kw_gs'
+GOOGLE_SHEET_URL_TABLE = 'gdrive.bi.marketing_itaib_semrush_url_gs'
+GOOGLE_SHEET_DB_TABLE = 'gdrive.bi.marketing_itaib_semrush_database_gs'
+
+PRESTO_KW_LIST = 'sandbox.itaib.seo_kw_semrush'
+PRESTO_URL_LIST = 'sandbox.itaib.seo_url_semrush'
+PRESTO_DB_LIST = 'sandbox.itaib.seo_databases_semrush'
+
+BQ_DELTA_TABLE_SEMRUSH_KW_TRAFFIC = 'temp.semrush_kw_traffic_delta'
+BQ_DELTA_TABLE_SEMRUSH_URL_BACKLINKS = 'temp.semrush_url_backlinks_delta'
+BQ_DELTA_TABLE_SEMRUSH_KW_SERP_FEATURES_MAP = 'temp.semrush_kw_serp_features_map_delta'
 
 # Update input tables
-script = func.update_input_tables()
-func.execute_sql(sql_script=script)
+pc = WixPrestoConnection()
+script_kw_list = func.update_input_tables(create_presto_table=PRESTO_KW_LIST, exesting_google_sheet_table=GOOGLE_SHEET_KW_TABLE)
+func.execute_sql(sql_script=script_kw_list, pc=pc)
+script_url_list = func.update_input_tables(create_presto_table=PRESTO_URL_LIST, exesting_google_sheet_table=GOOGLE_SHEET_URL_TABLE)
+func.execute_sql(sql_script=script_kw_list, pc=pc)
+script_db_list = func.update_input_tables(create_presto_table=PRESTO_DB_LIST, exesting_google_sheet_table=GOOGLE_SHEET_DB_TABLE)
+func.execute_sql(sql_script=script_kw_list, pc=pc)
 
 # Get input tables
-db_list = func.get_quix_data('select * from sandbox.itaib.seo_databases_semrush').database.tolist()
-kw_list = func.get_quix_data('select * from sandbox.itaib.seo_kw_semrush limit 2').keyword.tolist()  # todo: remove 'limit 10' when script is ready
-url_list = func.get_quix_data('select * from sandbox.itaib.seo_url_semrush limit 2').url.tolist()  # todo: remove 'limit 10' when script is ready
+KW_LIST = func.get_quix_data(f'select * from {PRESTO_KW_LIST}', pc=pc).keyword.tolist()

```

Comment:
why do you use execute_sql_pandas in get_quix_data and then convert it into list instead of using execute_sql ?